tests/forms_tests/tests/test_forms.py

```diff
@@ -2326,6 +2326,34 @@ class UserRegistration(Form):
 <input type="hidden" name="next" value="/"></li>"""
         )
 
+    def test_help_text_is_escaped(self):
```

Comment:
```suggestion
    def test_help_text_html_safe(self):
```

Reply:
Yes, I got my this phrasing here the wrong way round. Reading the test name again, I'm not sure it's that obvious what is going on here. Maybe we should add a doc string? I could also be overthinking it at this point. 
```python
"""
help_text is provided by developers and therefore should not be escaped. See ticket #6041.
"""
```

Reply:
I added a short docstring but without `is provided by developers ...`, this is not a strong argument, a lot of things are provided by developers but we don't mark them as safe. I'd say that's how it is, the decision was made a lot time ago :shrug: